helpers/accessServices.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable require-jsdoc */
```

Comment:
Why did you disable JSDOC. Is there any reason?

Reply:
@ericnyirimana Already corrected